### PR TITLE
PSAPC-217 create capture functionality

### PR DESCRIPTION
**Pull request by rene-gust**

### CODE REVIEW

File: src/Controller/Admin/OrderOverview.php [hunk 1]
```diff
@@ -140,4 +143,47 @@ public function refundpayment()
             );
         }
     }
+
+    public function makecharge()
```
Comment (rene-gust) [inline src/Controller/Admin/OrderOverview.php:147]: imho we should use came case for function names

File: src/Controller/Admin/OrderOverview.php [hunk 2]
```diff
@@ -140,4 +143,47 @@ public function refundpayment()
...
+            $repository = oxNew(LogRepository::class);
+            $logMessages = $repository->findLogMessageForOrderId($this->getEditObjectId());
+            if (!empty($logMessages)) {
+                foreach ($logMessages as $logMessage) {
+                    $logsWithChargePermission = $repository->findLogMessageForChargePermissionId(
+                        $logMessage['OSC_AMAZON_CHARGE_PERMISSION_ID']
+                    );
+                    foreach ($logsWithChargePermission as $logWithChargePermission) {
+                        if ($logWithChargePermission['OSC_AMAZON_RESPONSE_MSG'] === 'Captured') {
+                            return '-1';
```
Comment (rene-gust) [inline src/Controller/Admin/OrderOverview.php:174]: is it correct to return here instead of break?

File: src/Controller/Admin/OrderOverview.php [hunk 3]
```diff
@@ -140,4 +143,47 @@ public function refundpayment()
...
+                        }
+                        $chargeIdSet = isset($logWithChargePermission['OSC_AMAZON_CHARGE_ID'])
+                            && $logWithChargePermission['OSC_AMAZON_CHARGE_ID'] !== 'null';
+                        if ($chargeIdSet) {
+                            $chargeId = $logWithChargePermission['OSC_AMAZON_CHARGE_ID'];
+                            break;
```
Comment (rene-gust) [inline src/Controller/Admin/OrderOverview.php:180]: break or break 2?

File: src/Controller/Admin/OrderOverview.php (hunk 2 above)
Comment (HonkDerHase) [reply src/Controller/Admin/OrderOverview.php:174]: mario will fix the code smells

**Pull request merged**